builtin/src/lib.rs

```diff
@@ -2,16 +2,15 @@ use fil_actors_integration_tests::TEST_FAUCET_ADDR;
 use fvm_actor_utils::shared_blockstore::SharedMemoryBlockstore;
 use fvm_shared::{state::StateTreeVersion, version::NetworkVersion};
 use fvm_workbench_api::{bench::WorkbenchBuilder, wrangler::ExecutionWrangler};
-use fvm_workbench_vm::{
-    builder::FvmBenchBuilder, externs::FakeExterns, primitives::FakePrimitives,
-};
+use fvm_workbench_vm::{builder::FvmBenchBuilder, externs::FakeExterns};
 use genesis::{create_genesis_actors, GenesisSpec};
+use vm_api::MockPrimitives;
 
 pub mod genesis;
 
 /// Create an ExecutionWrangler with sensible genesis state and defaults for running imported
 /// tests from builtin-actors
-pub fn setup() -> ExecutionWrangler {
+pub fn setup(primitives: Box<dyn MockPrimitives>) -> ExecutionWrangler {
```

Comment:
I think it would be ok to revert this and create FakePrimitives here. All the call sites will be the same, and the real injecting of behaviour will happen when test code mutates MockPrimitives.